echopype/tests/echodata/test_echodata_combine.py

```diff
@@ -46,12 +52,30 @@ def ek80_test_data(test_path):
     return [test_path["EK80_NEW"].joinpath(*f) for f in files]
 
 
+@pytest.fixture
+def ek80_broadband_same_range_sample_test_data():
+    return ["s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205615.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205659.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH1707/EK80/D20170826-T205742.raw"]
+
+
+@pytest.fixture
+def ek80_narrowband_diff_range_sample_test_data():
+    return ["s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T130426.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T131325.raw",
+            "s3://noaa-wcsd-pds/data/raw/Bell_M._Shimada/SH2106/EK80/Hake-D20210701-T131621.raw"]
+
+
 @pytest.fixture
 def azfp_test_data(test_path):
+
+    # TODO: in the future we should replace these files with a remote pull, one
+    #  possible option is the data in
+    #  "https://rawdata.oceanobservatories.org/files/CE01ISSM/R00015/instrmts/dcl37/ZPLSC_sn55076/DATA/202109/*"
```

Comment:
```suggestion
    # TODO: in the future we should replace these files with another set of 
    # similarly small set of files, for example the files from the location below:
    #  "https://rawdata.oceanobservatories.org/files/CE01ISSM/R00015/instrmts/dcl37/ZPLSC_sn55076/DATA/202109/*"
    # This is because we have lost track of where the current files came from,
    # since the filenames does not contain the site identifier.
```